react-app/config-overrides.js

```diff
@@ -0,0 +1,25 @@
+module.exports = function override(config, env) {
```

Comment:
This file is currently necessary because `aem-react-editable-components` is not built for ES6. See https://github.com/adobe/aem-react-editable-components/pull/134